### ISSUE TITLE
Keptn CLI cant be used for automation due to Kube context check

### ISSUE THREAD

**Issue by grabnerandi** [keptn/keptn#3208]
With the current version of the Keptn CLI for 0.8 alpha the CLI always seems to make a check whether the Keptn CLI is authenticated against a keptn that runs on the current kubectl context. If that is the case the Keptn CLI prompts the user on whether to switch kube context or not with the following message:

```console
Kube context has been changed to xxxxxxxxx
Do you want to switch to the new Kube context with te Keptn running there? (y/n)
```

I think there are two issues with this current behavior
1. Keptn CLI for most operations interacts with Keptn via the API and doesnt need the Kube Context - so - why are we checking Kube Context if the CLI doesnt need to access k8s?
2. The answer to the prompt cant be automated right now - which makes it hard to automate the Keptn CLI

**Environment**
* Client OS (e.g., Linux, OSX, Windows):
* Keptn Version (`keptn version`): 0.8alpha

**Affected Component**
* [x] CLI

![image](https://user-images.githubusercontent.com/675886/107346095-9ae89480-6ac4-11eb-8657-484063bb282b.png)

**Comment by christian-kreuzberger-dtx** [comment]
> Keptn CLI for most operations interacts with Keptn via the API and doesnt need the Kube Context - so - why are we checking Kube Context if the CLI doesnt need to access k8s?
One of the reasons that we have seen is that people expect the `keptn` CLI to work similar like `helm` and other CLIs known in the Kubernetes world, as they always connect to the current Kubernetes context.

> The answer to the prompt cant be automated right now - which makes it hard to automate the Keptn CLI
I haven't seen this problem with our integration tests, but I'm happy to discuss the problem you run into with this.
I think this problem really just exists when you're working with different Kube contexts.

Nevertheless, I agree that there needs to be an option to hide this prompt.

In code, this is the part where we can interact:
https://github.com/keptn/keptn/blob/15a83681e5efe49f33adf736e0c9623dc836d6d9/cli/pkg/credentialmanager/handler.go#L142-L153

Most notably, setting `GlobalCheckForContextChange` to `true` should do the trick.

I believe something like `--disable-kube-context-check` should be fine, but I'm happy to hear other opinions (pinging @ankitjain28may and @bacherfl )

**Comment by christian-kreuzberger-dtx** [comment]
FYI, in `keptn auth` we have support for automatically changing the Kube Context, e.g.:
```
	authCmd.Flags().BoolVarP(&authParams.acceptContext, "yes", "y", false, "Automatically accept change of Kubernetes Context")
```

Should this make it to our root command?

**Comment by christian-kreuzberger-dtx** [comment]
As I'm not able to complete this task without answers provided and I'm going to be out-of-office for a couple of days, I'm unassigning myself and putting it back into the backlog.

**Comment by ankitjain28may** [comment]
>One of the reasons that we have seen is that people expect the keptn CLI to work similar like helm and other CLIs known in the Kubernetes world, as they always connect to the current Kubernetes context.

Yes, this is correct, people want to use it similar to any other CLI since keptn CLI talks to keptn API, it kept connected to the previous KEPTN installation even if we changed the context that's why, we have added the support, which asks the user to switch to the current context keptn installation on context switch. 

we have this flag in auth command
```
 -y, --yes                      Automatically accept change of Kubernetes Context
``` 
for accepting the current context but yes, if required we can add `--disable-kube-context-check` flag to disable it globally.

**Comment by christian-kreuzberger-dtx** [comment]
Thanks Ankit!

I just verified that the bug exists, using the following:

1. Connect your Keptn Installation and Kubernetes Cluster (pre-req)
2. Verify that you are connected using `keptn status`
3. Spin up a new kubernetes cluster and set KUBECONFIG, e.g., using k3s
   ```
   curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION=v1.19.5+k3s1 K3S_KUBECONFIG_MODE="644" sh -s - --no-deploy=traefik
   export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
   ```
4. Run `keptn status`
   ![image](https://user-images.githubusercontent.com/56065213/108491550-5a4b0100-72a4-11eb-8eee-c2fc8ea278a9.png)

# Workaround

Use `KUBECONFIG="" keptn status`:
![image](https://user-images.githubusercontent.com/56065213/108491650-764ea280-72a4-11eb-898c-805214cfb5ff.png)

# Next Steps

As there is a workaround, we do not need to fix this in 0.8.0 (e.g., using `--disable-kube-context-check`), instead we can implement a proper enhancement in 0.8.1 (e.g., using `-y/--yes`).

- [ ] Add this as a known issue for 0.8.0 release @johannes-b 
- [ ] After a short discussion with @warber it makes sense to create an enhancement where we implement `-y/--yes` as a "assume positive response to every query" in the CLI (we can re-use this issue)

**Comment by christian-kreuzberger-dtx** [comment]
`keptn generate support-archive` suffers from the same problem and can barely be automated.

I've been doing it the following way: `echo "y" | keptn generate support-archive`
